packages/utilities/src/ExecUtils.ts

```diff
@@ -0,0 +1,43 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { SpawnSyncOptions } from "child_process";
+import * as spawn from "cross-spawn";
+
+/**
+ * A collection of utilities related to executing a sub-process.
+ * @export
+ * @class ExecUtils
+ */
+export class ExecUtils {
+    /**
+     * Spawn a process with arguments and throw an error if the process fails.
+     * Parameters are same as `child_process.spawnSync` (see Node.js docs).
+     * Use this method if you want the safe argument parsing of `spawnSync`
+     * combined with the smart output handling of `execSync`.
+     * @returns Contents of stdout as buffer or string
+     */
+    public static spawnAndGetOutput(command: string, args?: string[], options?: SpawnSyncOptions): Buffer | string {
```

Comment:
Should we have the deprecated `ProcessUtils.execAndCheckOutput` function call this one?
That may avoid some duplication.

Not a huge deal though 😋

Reply:
At first I thought that changing the automated tests would be much harder than changing the actual product, It turned out that changing the tests were pretty easy too, so I committed this change.